reframe/frontend/autodetect.py

```diff
@@ -280,7 +282,8 @@ def detect_topology():
                 _save_info(topo_file, part.processor.info)
             elif detect_remote_systems:
                 with runtime.temp_environment(modules=modules, env_vars=vars):
-                    part._processor = ProcessorInfo(_remote_detect(part))
+                    part._processor = ProcessorInfo(_remote_detect(
+                                                    part, job_options))
```

Comment:
Better reformat as follows; it's cleaner:

```suggestion
                    part._processor = ProcessorInfo(
                        _remote_detect(part, job_options)
                    )
```